packages/discord-bot/src/utils/praiseEmbeds.ts

```diff
@@ -262,3 +262,16 @@ export const selfPraiseWarning = async (): Promise<string> => {
   }
   return 'SELF-PRAISE NOT ALLOWED, PRAISE GIVERS UNABLE TO PRAISE THEMSELVES (message not set)';
 };
+
+/**
+ * Generate response info message FIRST_TIME_PRAISER
+ *
+ * @returns {Promise<string>}
+ */
+export const firstTimePraiserInfo = async (): Promise<string> => {
+  const msg = (await settingValue('FIRST_TIME_PRAISER')) as string;
+  if (msg) {
+    return msg;
+  }
+  return 'FIRST TIME PRAISER MESSAGE';
```

Comment:
Could we update this to a template message that gives some info that the user can interpret(like - 'YOU ARE PRAISING FOR THE FIRST TIME. WELCOME TO PRAISE!')?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Good call. Changed.